service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -285,7 +269,7 @@ private HearingBooking getFirstHearing(List<Element<HearingBooking>> hearingBook
     private Map<DirectionAssignee, List<Element<Direction>>> sortDirectionsByAssignee(List<Element<Direction>> list) {
         List<Element<Direction>> nonCustomDirections = commonDirectionService.removeCustomDirections(list);
 
-        return commonDirectionService.sortDirectionsByAssignee(nonCustomDirections);
+        return getMapping(nonCustomDirections);
```

Comment:
would getMappingOfDirectionToAssignee be a more clear method name? happy to leave if you think it's explanation enough. :)